.travis.yml

```diff
@@ -0,0 +1,11 @@
+language: objective-c  
+
+before_install:
+    - rvm use system
+    - sudo gem install cocoapods -v '1.7.1'
+    - pod repo update  
+
```

Comment:
I believe you miss the lint & building step

Reply:
I don't think so because the travis log said pod installation success and the SwiftLint will trigger while build in project "Build Phases" setting.

Currently I'm trying to fix the error log 
**"xcodebuild: error: Unable to find a destination matching the provided destination specifier: { platform:iOS Simulator, OS:11.3, name:iPhone X }"**.